scripts/wawCommons.py

```diff
@@ -411,9 +411,21 @@ def getParametersCombination(config, *args):
     
     return parametersCombinationMap
 
-def setLoggerConfig():
-    fileConfig(os.path.split(os.path.abspath(__file__))[0]+'/logging_config.ini')
+def verbose(self, message, *args, **kws):
+    if (logging.Logger.isVerbose):
```

Comment:
I would change this to `self.isVerbose` and add some explanation why this function has self as a parameter. Even that I would probably put one more `if` to check if the `self` is type of object that you expect (somebody could try to call it with his own object... or maybe some more comments or that explanation could be enough... but you can imagine while we have some useful function in wawCommons which could be used across all scripts then somebody new could consider as a good idea to use this function directly).

Reply:
Or maybe better solution could be to define this function directly inside `setLoggerConfig` to be sure that nobody will ever call it directly.

Reply:
- `isVerbose` has to be in the class as each script needs to have the same verbosity.
- `self` has to be a parameter because the function is set to be a method of the logger class.
- Moving the function inside `setLoggerConfig` is a good idea.

Fixed in 8705b2165b34e907726bbebe006a00b7ca40e531